models/movie.ts

```diff
@@ -65,11 +67,26 @@ export const Movie: ListConfig<Lists.Movie.TypeInfo<any>, any> = list({
       },
     }),
     howToWatch: text(),
-    handicap: integer({ defaultValue: 0 }),
-
     posts: relationship({ ref: "Post.movies", many: true }),
     sounds: relationship({ ref: "Sound.movies", many: true }),
     keywords: relationship({ ref: "Keyword.movies", many: true }),
+    handicap: virtual({
+      field: graphql.field({
+        type: graphql.Int,
+        async resolve(item, args, context) {
+          //@ts-ignore
```

Comment:
There is an error here from typescript that the type of context is unknown. I tried using the keystone context type and it threw more errors. Code is working, but it should be fixed eventually.